frontend/src/routes/groups/[group_id]/+page.svelte

```diff
@@ -26,8 +43,129 @@
 			title: 'Trail Maintenance'
 		}
 	];
+
+	const libre311 = useLibre311Service();
+
+	async function handleAddNewService() {
+		newServiceName = stringValidator(newServiceName);
+		newServiceCode = stringValidator(newServiceCode);
+		newServiceDescription = stringValidator(newServiceDescription);
+		newServiceDefinition = stringValidator(newServiceDefinition);
+		newServiceGroupId = numberValidator(newServiceGroupId);
+
+		const resultSet = new Set([
+			newServiceName.type,
+			newServiceCode.type,
+			newServiceDescription.type,
+			newServiceDefinition.type,
+			newServiceGroupId.type
+		]);
+		if (resultSet.has('invalid')) return;
+
+		console.log(
+			await libre311.createService({
+				service_code: newServiceCode.value ?? '',
```

Comment:
Service codes have to be unique across jurisdictions so if we default to an empty string it will work once and then start erroring out.